nano/node/common.cpp

```diff
@@ -996,6 +1011,17 @@ bool nano::frontier_req::operator== (nano::frontier_req const & other_a) const
 	return start == other_a.start && age == other_a.age && count == other_a.count;
 }
 
+std::string nano::frontier_req::to_string ()
```

Comment:
remove leading space, this could be compressed to one line without whitespace if you shorted the message.

E.g.
```
start=xxxx age=yyyy count=zzzz
```

Try not to avoid whitespace, ideally use bio vertical whitespace.
Use vertical whitespace only where it adds a lot of value.

Reply:
pushed just now